src/core/Notification/Notification.test.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { Notification } from './Notification';
+import { axeTest } from '../../utils/test';
+
+describe('Error view tests', () => {
+  const ErrorNotification = (
+    <Notification
+      status="error"
+      className="notification-test-class"
+      closeText="Close"
+      smallScreen
+      labelText="Lorem ipsum dolor sit"
+    >
+      Testcontent
+    </Notification>
+  );
+
+  it('should have status and smallScreen prop specific classNames', () => {
+    const { container } = render(ErrorNotification);
+    expect(container.firstChild).toHaveClass('fi-notification--error');
+    expect(container.firstChild).toHaveClass('fi-notification--small-screen');
+  });
```

Comment:
This would be better to change a little.

We are testing the `status` prop here with the **error** value.

So we want own `describe` block, named **status**, inside the existing props block like other tests in this file
And then appropriate name for the test that tells what kind of status variant we are testing.

Reply:
I don't see the reason for the `smallScreen` prop test to be here. It should have it's own describe block to test the effects of `smallScreen` prop.

Reply:
Fixed. Status and smallScreen have now own describe section

Reply:
They now seem to be in own blocks, but those new describe blocks are not inside the existing `props` block like other props (e.g children and className).

Other way to say, they are not nested like others. They should be nested inside the **props**.

```tsx
 describe('props', () => {
  // other existing ones
  describe('status', () => {
  // ...
  });
  describe('smallScreen', () => {
  // ...
  });
  // other existing ones
});
```

Reply:
Fixed. Those are now inside of props